core/chains/evm/config/toml/defaults/Scroll_Mainnet.toml

```diff
@@ -0,0 +1,21 @@
+ChainID = '534352'
+FinalityDepth = 1
+LogPollInterval = '3s'
+MinIncomingConfirmations = 1
+# Scroll only emits blocks when a new tx is received, so this method of liveness detection is not useful
+NoNewHeadsThreshold = '0'
+OCR.ContractConfirmations = 1
+
+[GasEstimator]
+Mode = 'L2Suggested'
+# Scroll uses the L2Suggested estimator; we don't want to place any limits on the minimum gas price
+PriceMin = '0'
+# Never bump gas on Scroll
+BumpThreshold = 0
+
+[GasEstimator.BlockHistory]
+# Force an error if someone enables the estimator by accident; we never want to run the block history estimator on Scroll
+BlockHistorySize = 0
+
+[HeadTracker]
+HistoryDepth = 50
```

Comment:
Just curious, why a history depth of 50, if finalityDepth = 1?

Reply:
The idea was to reduce the overhead of storing the history since the default is 100. 
Also was not sure if setting it too low would have any consequences, so used 50 similar to what's set here for Celo: https://github.com/smartcontractkit/chainlink/blob/develop/core/chains/evm/config/toml/defaults/Celo_Mainnet.toml#L19